packages/edit-site/src/components/site-hub/index.js

```diff
@@ -170,8 +166,7 @@ export const SiteHubMobile = memo(
 					<HStack>
 						<div className="edit-site-site-hub__title">
 							<Button
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
This can be seen when you visit and check for site title in site editor mode for mobile.

**Note:** It has link variant, hence 40px size does not affect the button.

| Before | After     |
| :-------- | :------- |
| <img width="419" alt="site-hub-mobile-title-before" src="https://github.com/user-attachments/assets/8375895d-846d-4c10-b720-37649196223b"> | <img width="419" alt="site-hub-mobile-title-after" src="https://github.com/user-attachments/assets/048d8e15-d469-4440-885f-82fc54d6604a"> |